src/embeddings/embeddings_api.cpp

```diff
@@ -31,9 +31,17 @@
 
 using namespace rapidjson;
 
+enum class InputType {
+    NONE,
+    STRING,
+    INT,
+    INT_VEC
+};
+
```

Comment:
```suggestion
enum class InputType {
    NONE,
    STRING,
    TOKENS
};

I don't understand why we do have different type for single batch tokens and multi batch tokens but we do not have one for single string (1 document) and multiple documents (1+ documents). Isn't 1+ strings not supported?

```

Reply:
In this case single string is represented as vector with size of one and is sharing the same path as when there is multiple strings in the input request